react-client/src/style.css

```diff
@@ -108,27 +106,15 @@
   margin-right: 10px;
   position: relative;
   height: 55px;
-  font-size: 16;
+  font-size: 14.4px;
```

Comment:
You might want to use em instead of px. Some explanation - https://medium.com/code-better/css-units-for-font-size-px-em-rem-79f7e592bb97